pkg/operator/ceph/file/subvolumegroup/controller_test.go

```diff
@@ -46,6 +46,7 @@ func TestCephClientController(t *testing.T) {
 	// Set DEBUG logging
 	capnslog.SetGlobalLogLevel(capnslog.DEBUG)
 	os.Setenv("ROOK_LOG_LEVEL", "DEBUG")
+	os.Setenv("CSI_ENABLE_HOST_NETWORK", "true")
```

Comment:
try switching to `t.SetEnv` instead of `os.SetEnv` in the test files as it takes care of resetting the values once the testing is done.